### ISSUE TITLE
Support code fold

### ISSUE THREAD

**Issue by dingyi222666** [Rosemoe/sora-editor#85]

**Comment by Rosemoe** [comment]
@dingyi222666 It's hard to implement so it has a low priority.

**Comment by Arashvscode** [comment]
> @dingyi222666 It's hard to implement so it has a low priority.

I don't think it will be difficult to search the Google source and view the complete source of Android Studio, which has been public for several years!

**Comment by Rosemoe** [comment]
> it's interesting and important to the whole ide. Are you going to do it?

Yes. We'll implement code folding in future.

**Comment by AYssu** [comment]
代码折叠有最新的进度吗，我觉得这个的作用还是很大的，减少Android 小屏幕的视觉疲惫和大幅度滑动

**Comment by Rosemoe** [comment]
> 代码折叠有最新的进度吗，我觉得这个的作用还是很大的，减少Android 小屏幕的视觉疲惫和大幅度滑动

目前的计划是先完成 #767 的内联提示之后，再考虑做代码折叠。

**Comment by Ghostreal30** [comment]
Weak humans 😂😂😂😂
